getpaid/tests.py

```diff
@@ -148,6 +144,43 @@ def test_online_payment_failure(self):
         payment = Payment.objects.get(pk=payment.pk)
         self.assertEqual(payment.status, 'failed')
 
+    def test_online_payment_rejection(self):
+        """Ensure that order may reject the payment."""
+        Payment = get_model('getpaid', 'Payment')
+        order = Order(name='Test EUR order', total='123.45', currency='PLN')
```

Comment:
You’re right, this is mixed up in the whole suite, so reproducing it here maintains some consistency. ;) I can fix it in this PR if it is desirable.